front-packages/shared/src/components/DoubleCheckDeleteModal.tsx

```diff
@@ -18,10 +18,18 @@ const DoubleCheckDeleteModal = ({
   const [value, setValue] = useState('');
   const textIsConfirmed = value === textToCheck;
 
+  const handleConfirm = () => {
+    if (!textIsConfirmed) {
```

Comment:
In order to avoid double submit 
```suggestion
    if (!canConfirmDelete || !textIsConfirmed) {
```